container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/create_engine.go

```diff
@@ -29,8 +29,8 @@ const (
 	enclaveManagerUIPort                        = 9711
 	enclaveManagerAPIPort                       = 8081
 	engineDebugServerPort                       = 50102 // in ClI this is 50101 and 50103 for the APIC
-	maxWaitForEngineAvailabilityRetries         = 10
-	timeBetweenWaitForEngineAvailabilityRetries = 1 * time.Second
+	maxWaitForEngineAvailabilityRetries         = 40
```

Comment:
It's increasing the timeout because now the engine could take more time to restart if it has to restart several APIC containers.
I've tested it with 20 APIC containers running at the same time